airflow/providers/amazon/aws/example_dags/example_eks_with_nodegroups.py

```diff
@@ -40,11 +39,15 @@
     'endpointPrivateAccess': False,
 }
 
+default_args = {
+    'cluster_name': CLUSTER_NAME,
+}
 
 with DAG(
     dag_id='example_eks_with_nodegroups_dag',
+    default_args=default_args,
```

Comment:
Same comment here too. Thanks for adding these in! 🚀

Reply:
Should be addressed in https://github.com/apache/airflow/pull/18645/commits/b577f159075d1c6b75d90c54303041b78025f122